### PR TITLE
[iCloud binding] Update help for things and change the transformation for items

### PR DESCRIPTION
**Pull request by bredmich**
Adding informations about the device ID and location information in the .things file.
- It´s clear where to find the device ID. I had so search in the community to found someone mentioning where to find the device ID.

Changing the transformation for 'batteryStatus' and 'batteryLevel'.
- The transformation [%s %% ] for 'batteryStatus' is wrong and brings the information "Not charging %"
- The transformation [%.0f] for 'batteryLevel' is wrong and shows the battery level without '%'

### CODE REVIEW

**Comment by martinvw** [comment]
@pgfeller

File: addons/binding/org.openhab.binding.icloud/README.md [hunk 1]
```diff
@@ -67,13 +67,17 @@ Bridge icloud:account:myaccount [appleId="mail@example.com", password="secure",
 }
 ```
 
+The device ID can be found in the Paper UI inbox.
+The information '@ "World"' is optional.
+
+
```
Comment (Confectrician) [inline addons/binding/org.openhab.binding.icloud/README.md]: One empty line should be removed.

File: addons/binding/org.openhab.binding.icloud/README.md [hunk 2]
```diff
@@ -67,13 +67,17 @@ Bridge icloud:account:myaccount [appleId="mail@example.com", password="secure",
 }
 ```
 
+The device ID can be found in the Paper UI inbox.
+The information '@ "World"' is optional.
```
Comment (Confectrician) [inline addons/binding/org.openhab.binding.icloud/README.md]: *italic* would be better for highlighting the @ World

`*@ "World"*`

File: addons/binding/org.openhab.binding.icloud/README.md (hunk 1 above)
Comment (bredmich) [reply addons/binding/org.openhab.binding.icloud/README.md]: removed

File: addons/binding/org.openhab.binding.icloud/README.md (hunk 2 above)
Comment (bredmich) [reply addons/binding/org.openhab.binding.icloud/README.md]: changed to italic

**Comment by martinvw** [comment]
It would be great if you could include a `Signed-off-by` (to certify that you wrote it or otherwise have the right to pass it on as an open-source), see http://docs.openhab.org/developers/contributing/contributing.html#sign-your-work

If running a local git client you can use `git commit --amend --signoff`

**Comment by martinvw** [comment]
@bredmich @Confectrician I think its fine but I would appreciate @pgfeller opinion as well since he wrote this binding.

**Comment by martinvw** [comment]
@pgfeller please :-)

**Comment by martinvw** [comment]
@bredmich it would be great if you could fix the signing off

**Comment by bredmich** [comment]
Hi Martin,
As i‘m only using Github online i‘m not able to sign-off anymore.
I could only create a new PR.

Kind regards

**Comment by bredmich** [comment]
Signed-off-by: Michael Bredehorn <michael@bredehorn.nrw> (github: bredmich)

**Comment by martinvw** [comment]
Thanks!

**Pull request merged**